src/arrayMethodSort.js

```diff
@@ -3,9 +3,19 @@
 /**
  * Implement method Sort
  */
+const defaultCompare = (a, b) => a.toString() > b.toString();
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = defaultCompare) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
```

Comment:
The compareFunction should return a number, but the '>' operator will return a boolean. You should be comparing the result of the compareFunction to 0, not to a boolean value.